pkg/operators/otel-profiles/otel-profiles.go

```diff
@@ -0,0 +1,518 @@
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/datasource"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/gadget-service/api"
+	apihelpers "github.com/inspektor-gadget/inspektor-gadget/pkg/gadget-service/api-helpers"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/operators"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/operators/otel-profiles/orderedset"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/params"
+)
+
+const (
+	opName = "otel-profiles"
+
+	ParamOtelProfilesExporter = "otel-profiles-exporter"
+
+	ExporterOTLPGRPC = "otlp-grpc"
+
+	CompressionNone = "none"
+	CompressionGZIP = "gzip"
+
+	stackFieldsAnnotation     = "profiles.stack-fields"
+	valueFieldAnnotation      = "profiles.value-field"
+	sampleAttributeAnnotation = "profiles.sample-attribute"
+	profilesTypeAnnotation    = "profiles.type"
+	profilesUnitAnnotation    = "profiles.unit"
+)
+
+var supportedExporters = []string{ExporterOTLPGRPC}
+
+type profileConfig struct {
+	Exporter    string `json:"exporter" yaml:"exporter"`
+	Endpoint    string `json:"endpoint" yaml:"endpoint"`
+	Insecure    bool   `json:"insecure" yaml:"insecure"`
+	Compression string `json:"compression" yaml:"compression"`
+}
+
+type otelProfilesOperator struct {
+	clients  map[string]pprofileotlp.GRPCClient
+	callOpts []grpc.CallOption
+}
+
+func (o *otelProfilesOperator) Name() string {
+	return opName
+}
+
+func (o *otelProfilesOperator) Init(params *params.Params) error {
+	if config.Config == nil {
+		return nil
+	}
+
+	o.clients = make(map[string]pprofileotlp.GRPCClient)
+	o.callOpts = make([]grpc.CallOption, 0)
+
+	configs := make(map[string]*profileConfig, 0)
+	err := config.Config.UnmarshalKey("operator.otel-profiles.exporters", &configs)
+	if err != nil {
+		log.Warnf("failed to load operator.otel-profiles.exporters: %v", err)
+	}
+
+	for k, v := range configs {
+		if v.Exporter != ExporterOTLPGRPC {
+			return fmt.Errorf("unsupported log exporter %q; expected one of %s", v.Exporter,
+				strings.Join(supportedExporters, ", "))
+		}
+		var options []grpc.DialOption
+
+		if v.Insecure {
+			options = append(options, grpc.WithTransportCredentials(insecure.NewCredentials()))
+		}
+		switch v.Compression {
+		default:
+			return fmt.Errorf("unsupported log compression %q", v.Compression)
+		case "", CompressionNone:
+		case CompressionGZIP:
+			o.callOpts = append(o.callOpts, grpc.UseCompressor(gzip.Name))
+		}
+
+		conn, err := grpc.NewClient(v.Endpoint, options...)
+		if err != nil {
+			log.Errorf("creating gRPC connection to %q: %v", v.Endpoint, err)
+			return fmt.Errorf("creating gRPC connection to %q: %w", v.Endpoint, err)
+		}
+
+		o.clients[k] = pprofileotlp.NewGRPCClient(conn)
+	}
+
+	return nil
+}
+
+func (o *otelProfilesOperator) GlobalParams() api.Params {
+	return api.Params{}
+}
+
+func (o *otelProfilesOperator) InstanceParams() api.Params {
+	return api.Params{
+		&api.Param{
+			Key:         ParamOtelProfilesExporter,
```

Comment:
Can you please add `Title` and `Tags` with a new `TagGroupOtelProfiles` (like [here](https://github.com/inspektor-gadget/inspektor-gadget/blob/main/pkg/operators/otel-logs/otel-logs.go#L135-L139))